lib/modules/dosomething/dosomething_organ_donation/dosomething_organ_donation.module

```diff
@@ -127,14 +127,18 @@ function dosomething_organ_donation_send_postal_code() {
  *
  */
 function dosomething_organ_donation_store_registration() {
+  // Grab this to get the sid
   $params = drupal_get_query_parameters();
 
+  // Grab this to get the uid
+  global $user;
+
   try {
-    if ($params['sid'] && $params['uid']) {
+    if ($params['sid'] && $user->uid) {
```

Comment:
@sbsmith86 yeah that seems like a good idea. Should that get added in [here](https://github.com/DoSomething/phoenix/blob/df9950e89038a155ae365ecdc69d5a6fa0aab3fc/lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc#L220-L236) as well or somewhere else?